content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+---
+title: API Server Flow Control
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The{{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
```

Comment:
I think we should stick to one user-facing name for this new feature (and also make all the code to use it, but that is not relevant to this PR).  I think we have converged on the name "API Priority and Fairness" for the feature.  Including "API" in the name is important because there are other places in the system where priority and fairness could conceivably appear.

Reply:
+1. should fix it to "API Priority and Fairness". we had several names in the past. neither of e.g. "APIRequestManagement", "Priority and Fairness", "flow-control" or "rate-limitting" should appear in the doc for clarification..